setup.py

```diff
@@ -41,7 +41,7 @@
         # The following are for Scenario Studio
         "yattag>=1.14.0",
         # The following is for both SS and Envision
-        "cloudpickle>=1.3.0,<1.4.0",
+        "cloudpickle>=1.3.0,<=2.1.0",
```

Comment:
What was the reason for constraining cloudpickle to between 1.3.0 and 2.1.0?

Reply:
I have no idea, I tried it out and had no issues (there may have been a bug in a version.) But considering there were no issues it looks like it was just constrained that way.